### ISSUE TITLE
implemented in tensorflow

### ISSUE THREAD

**Issue by zjjzby** [jlezama/OrthogonalLowrankEmbedding#2]
Hello! I wanna apply the OLE loss in tensorflow. Do you have a plan to develop tensorflow version of  implementation? Thanks!

**Comment by jlezama** [comment]
Hello and thanks a lot for your interest.
We would love to see a tensorflow version but we are not working on it yet.
However the currently available implementation is basically Numpy so it should be easy to adapt ([direct link](https://github.com/jlezama/OrthogonalLowrankEmbedding/blob/master/pytorch_OLE/OLE.py)).
